docs: fix admin.path reserved paths typedoc and add standalone build guidance (#15193)
medusajs/medusa: www/apps/resources/app/medusa-cli/commands/build/page.mdx

By default, the Medusa Admin is built to the `.medusa/server/public/admin` directory.

If you want a separate build to host the admin as a standalone application, such as on Vercel, pass the `--admin-only` option as explained in the [Options](#options) section. This outputs the admin to the `.medusa/admin` directory instead.

> <Note>

Reviewer: ```suggestion
### Change Admin Base Path
```

> When deploying the admin as a standalone application on a separate domain, it will be served at the path set in `admin.path` (default: `/app`). To serve it at the root instead, set `admin.path` to `/` in `medusa-config.ts`:

Reviewer: ```suggestion
By default, the Medusa Admin dashboard is hosted at the `/app` path. You can change it with the `admin.path` configuration in `medusa-config.ts`.

For example, to serve it from the root path `/`:
```

```ts title="medusa-config.ts"
module.exports = defineConfig({
  admin: {
    path: "/",
    backendUrl: process.env.MEDUSA_BACKEND_URL,
  },
  // ...
})
```

`/` is a valid value for `admin.path` — it is not a reserved path.

> </Note>

Reviewer: ```suggestion
The admin will now run at `/` of your application. For example, at `acme.com`.
```
